inst/include/common/data_object.hpp

```diff
@@ -164,7 +163,7 @@ struct DataObject : public fims::FIMSObject<Type> {
    * @param i
    * @param j
    * @param k
-   * @return
+   * @return reference type
```

Comment:
Should this be `@return type`? Same comment for other lines in this file.

Reply:
what is actually returned is the data value at the defined index location. I looked up a similar type of function in C++ and found a more informative description: 
a reference to the value of the array at position i, j, k